contrib/clojure-package/README.md

```diff
@@ -147,7 +147,7 @@ The first step is to recursively clone the MXNet repository and checkout the des
   ```bash
   git clone --recursive https://github.com/apache/incubator-mxnet.git ~/mxnet
   cd ~/mxnet
-  git checkout tags/1.4.0 -b my_mxnet  # this is optional
+  git checkout tags/1.5.0 -b my_mxnet  # this is optional
```

Comment:
this needs to be 1.4.0

Reply:
Neither 1.4.0 nor 1.5.0 tag exists. 1.4.0 tag will point to a release in v1.4.x branch. Which should be the correct tag? Should it point to a release tag?

Reply:
It should point to the tag that corresponds to the jar on maven and the jars on maven are only for the last release. I'm assuming that it will be the 1.4.0 tag when this branch is merged (at least that's what I've been doing).  For reference when I do `git tag --list` I see:

```
1.3.1
1.3.1.rc0
```
I'm not exactly sure of the timing of when this PR is merged though .. is that a correct assumption?